jbmc/regression/jbmc-strings/StringConcat/test_string_nondet_array.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+Test.class
+--function Test.fromNonDetArray --depth 10000 --unwind 5 --throw-runtime-exceptions  --cp `../../../../scripts/format_classpath.sh . ../../../lib/java-models-library/target/core-models.jar`
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED
+line 318 no uncaught exception: FAILURE
```

Comment:
Can you check that the right exception is thrown?

Reply:
We could do that, if we didn't use `--throw-runtime-exceptions` but the failure I observed happened specifically when the option was used so I prefer to keep it. We could look for a particular exception in the trace, but here both NullPointerException and ArrayOutOfBoundsException could be valid counterexample traces.

Reply:
Those two exceptions could still be matched by a single regular expression?